src/main/java/org/mule/module/apikit/model/Entity.java

```diff
@@ -93,6 +89,19 @@ public List<String> getKeys() {
   }
 
   public static String pluralizeName(String name) {
-    return (name.endsWith("s") ? name : NAME_CONVERTER.pluralize(name));
+    return (name.endsWith("s") ? name : pluralize(name));
+  }
+
+  private static String pluralize(String name) {
+    if (!name.endsWith("List") && (!name.endsWith("s") || name.endsWith("ss"))) {
+      if (name.endsWith("y") && !name.endsWith("ay") && !name.endsWith("ey") && !name.endsWith("iy")
+          && !name.endsWith("oy") && !name.endsWith("uy")) {
+        return name.equalsIgnoreCase("any") ? name : name.substring(0, name.length() - 1) + "ies";
+      } else {
+        return name.endsWith("ss") ? name + "es" : name + 's';
+      }
+    } else {
+      return name;
+    }
```

Comment:
_i think_ this is equivalent.
```suggestion
    if (name.endsWith("s") || name.endsWith("List")) {
      return name;
    }
    if (name.endsWith("y") && !name.endsWith("ay") && !name.endsWith("ey") && !name.endsWith("iy")
        && !name.endsWith("oy") && !name.endsWith("uy")) {
      return name.equalsIgnoreCase("any") ? name : name.substring(0, name.length() - 1) + "ies";
    }
    return name + 's';
```